Traceback (most recent call last):
  File "C:\Users\Asus\Documents\Python\ai-model\app\app.py", line 20, in <module>
    model = joblib.load(model_path)
  File "C:\Users\Asus\Documents\Python\ai-model\venv\lib\site-packages\joblib\numpy_pickle.py", line 597, in load
    with open(filename, 'rb') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\Asus\\Documents\\Python\\ai-model\\models\\chatbot_model.joblib'

Failed to load responses.json:
Traceback (most recent call last):
  File "C:\Users\Asus\Documents\Python\ai-model\app\app.py", line 30, in <module>
    with open(os.path.join(project_root, 'responses.json'), 'r', encoding='utf-8') as f:
FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\Asus\\Documents\\Python\\ai-model\\responses.json'
